### PR TITLE
fix: Single phase kernels - split into separate files and fix a bug related to mass and energy updates for poromechanics

### PR DESCRIPTION
**Pull request by paveltomin**
(didn't touch hybrid FVM and proppant)

### CODE REVIEW

**Comment by paveltomin** [comment]
need review please

**Comment by paveltomin** [comment]
need reviews please

**Comment by paveltomin** [comment]
@CusiniM, @castelletto1, @corbett5, @dkachuma, @joshua-white, @rrsettgast, and/or @wrtobin need review please

**Review by corbett5 — APPROVED** [review]
LGTM

File: src/coreComponents/physicsSolvers/multiphysics/poromechanicsKernels/SinglePhasePoromechanicsConformingFractures.hpp [hunk 1]
```diff
@@ -31,7 +31,7 @@ namespace singlePhasePoromechanicsConformingFracturesKernels
 {
 
 template< integer NUM_EQN, integer NUM_DOF >
-class ConnectorBasedAssemblyKernel : public singlePhaseFVMKernels::FaceBasedAssemblyKernel< NUM_EQN, NUM_DOF, SurfaceElementStencilWrapper >
+class ConnectorBasedAssemblyKernel : public singlePhaseFVMKernels::FluxComputeKernel< NUM_EQN, NUM_DOF, SurfaceElementStencilWrapper >
```
Comment (CusiniM) [inline src/coreComponents/physicsSolvers/multiphysics/poromechanicsKernels/SinglePhasePoromechanicsConformingFractures.hpp:34]: I would have almost expected `ConnectorBasedAssemblyKernel` to be the base class for all `FluxAssemblyKernels`. I do like this name signficantly better than FaceBasedAssemblyKernel coz it's also a lot more general.

**Review by CusiniM — APPROVED** [review]

**Pull request merged**